docs/components/RemoteCacheCounter.tsx

```diff
@@ -8,37 +8,44 @@ const counterFormatter = Intl.NumberFormat(undefined, {
   maximumFractionDigits: 0,
 });
 
+const START_NUM = 2080300;
```

Comment:
Explanation of this number would be appreciated, it looks very magic to me.

Reply:
Oops, deleted the comment when I was messing around and forgot to bring it back. Thanks.